src/main/java/sirius/biz/importer/format/ImportDictionary.java

```diff
@@ -607,8 +608,7 @@ protected Optional<FieldDefinition> resolveComputedField(String name) {
      * @return the normalized version of the field
      */
     protected static String normalize(String field) {
-        return Strings.reduceCharacters(field.toLowerCase())
-                    .replaceAll("[^\\p{L}\\d_]", "");
+        return StringCleanup.reduceCharacters(field.toLowerCase()).replaceAll("[^\\p{L}\\d_]", "");
```

Comment:
This should better match the intended cleanup call structure.
Please check the suggestion for validity in the IDE before committing.
```suggestion
        return Strings.cleanup(field, StringCleanup::lowercase, StringCleanup::reduceCharacters, text -> text.replaceAll("[^\\p{L}\\d_]", ""));
```